dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -250,7 +250,8 @@
         "PiiEntityRecognition",
         "KeyPhraseExtraction",
         "LanguageDetection",
-        "EntityLinking"
+        "EntityLinking",
+        "ZeroShotClassification"
```

Comment:
Just want to confirm, is this a sync task only?

Reply:
yes